terraform/eks.tf

```diff
@@ -16,18 +16,23 @@ module "eks-production" {
       groups   = ["system:masters"]
     },
   ]
-  worker_groups_launch_template = [
+  map_users = [
     {
-      name                    = "spot-1"
-      override_instance_types = ["t3.medium"]
-      spot_instance_pools     = 1
```

Comment:
What are spot instance pools & why do we not need to specify this anymore? Also I noticed we didn't have `spot_allocation_strategy`, is it because we only had one pool before? 

Explanations I found
> A Spot Instance pool is a set of unused EC2 instances with the same instance type (for example, m5.large), operating system, Availability Zone, and network platform.

> In this case, the parameter `spot_instance_pools` is a part of the Spot allocation strategy and is used to define the number of Spot Instance pools to diversify the EC2 Auto Scaling group across in each Availability Zone.

Reply:
Actually wait is the reason we don't need to specify this anymore because managed node groups automate the provisioning of spot instance pools?

Reply:
Yep, the reason we don't need to manually specific a spot instance launch template is because we switched over to AWS managed groups where we just need to tell AWS we want spot instances (line 33)